opus/application/static_media/js/opus.js

```diff
@@ -617,8 +617,9 @@ var opus = {
         let adjustProductInfoHeightDB = _.debounce(o_cart.adjustProductInfoHeight, 200);
         let adjustDetailHeightDB = _.debounce(o_detail.adjustDetailHeight, 200);
         let adjustHelpPanelHeightDB = _.debounce(opus.adjustHelpPanelHeight, 200);
-        let adjustMetadataSelectorMenuPSDB = _.debounce(o_browse.adjustMetadataSelectorMenuPS, 200);
-        let adjustSelectedMetadataPSDB = _.debounce(o_browse.adjustSelectedMetadataPS, 200);
+        let adjustSelectMetadataHeightDB = _.debounce(o_browse.adjustSelectMetadataHeight, 200);
```

Comment:
Just a note: we can use non-debounced version ```let adjustSelectMetadataHeightDB = o_browse.adjustSelectMetadataHeight;``` to remove the jumping when browser width is changing around 1000px & 1001px, 775px & 774px, and 652px & 653px.

Reply:
Changed